posthog/tasks/tasks.py

```diff
@@ -33,7 +33,12 @@ def redis_heartbeat() -> None:
 
 @shared_task(ignore_result=True, queue=CeleryQueue.ANALYTICS_QUERIES.value)
 def process_query_task(
-    team_id: str, query_id: str, query_json: Any, limit_context: Any = None, refresh_requested: bool = False
+    team_id: str,
+    user_id: str,
```

Comment:
This _might_ be tricky in terms of deployment, in that the number of arguments to the task changes. AFAIK this will mean Celery raising a `TypeError` when processing a task queued pre-change on a post-change worker ([Celery docs](https://docs.celeryq.dev/en/stable/reference/celery.app.task.html)). Is the rollout significant enough for this to disrupt anyone though?
Looking just at the async queries feature flag, more care doesn't seem worth the effort. Unless there are places that rely on async queries other than `/query`